releases/release-1.23/release-notes/maps/pr-104699-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 104699
+releasenote:
+  text: When creating an object with generateName, if a conflict occurs the server
+    now returns an AlreadyExists error with a retry option.
```

Comment:
Not quite clear to me. Suggest: If a conflict occurs when creating an object with generateName, the server now returns an "already exists" error with a retry option instead of timeout.